### DISCUSSION TITLE
Readings in prep for week 10

### DISCUSSION THREAD

**Discussion by benmiller314** [benmiller314/dsam2023fall#13] [Reading responses etc]
After reading [Jaffe's short guide](https://guides.library.ucsc.edu/c.php?g=618773) to metadata creation, [Gilliland's theoretical framework](http://www.getty.edu/publications/intrometadata) of metadata varieties, the [Dublin Core's "beginner's guide"](https://www.dublincore.org/specifications/dublin-core/usageguide/2005-08-15/elements) to the 15 basic elements of that ontology, and [Jessop's reflections on selecting metadata fields](https://doi.org/10.2352/issn.2168-3204.2006.3.1.art00020) for several DH projects – and after at least glancing at the [alphabet soup of metadata standards](https://scalar.usc.edu/works/intro-to-dh-hs3393/media/seeingstandards_glossary_poster.pdf) aggregated by Riley – **please write a short response to let us know where your mind went**. For example, you might...

* Highlight new ideas or key terms, especially those that you expect will be useful in thinking about digital projects.
* Highlight sentences, passages, or methods/approaches that struck you as especially well done, perhaps worth imitating.
* Link outward to something a passage reminds you of, explaining where you're leading us.
* Pose questions to your classmates about a passage, especially (but not exclusively) if something is confusing.
* Respond to any of the above – which you can now do directly, using threads.

Where possible, please try to point us to at least one particular passage or chunk from the text you're responding to – quoting if you can, giving a timestamp or page reference either way – as a way to help keep our discussion grounded.

**Comment by chloe-cmu** [comment]
The discussion of metadata in general has pretty much gelled with our previous discussion of data -- which makes sense, as one of the readings defines metadata as "deliberate, structured data about data." (UC Santa Cruz) 

But I was intrigued by the idea of the importance of metadata in web searches. I ended up clicking through to the full chapter. I was particularly interested in the idea that internet searches are not actually "live."
The line about searching: "when users do a search on Google, they are not searching the web “live”—they are searching a vast corpus of tags and text that the search engine has stored in the form of an index; this is why a search via a commercial search engine sometimes results in a “page not found” message—that is, the page that was indexed by the web crawler is no longer available, but the data taken from it still exists in an index on one of the search provider’s servers." This was mostly interesting to me because it helps illuminate the black box of digital work a bit more. The Internet still seems more like magic than anything else to me in some ways, and so learning about how it actually works and what's happening is helpful. Google isn't magic; it's just doing something really complicated and challenging (and getting more complicated all the time). The chapter sums up Google's key actions as a search engine. It must maintain an comprehensive and current list of all publicly available web pages; maintain an index of that material (and thus the ability to search quickly); and be able to respond to millions of requests per day. As the reading points out, that's energy and labor intensive. I'm looking forward to learning more about the physicality of data in the next few weeks.

**Comment by SidraNArshad** [comment]
I read “Library Guide: Metadata Creation” with my and Khushboo’s projects in mind, as I think that the concerns we have regarding our projects align, and I think this is a great resource as we try to articulate our struggles and address them. I would say that, in terms of categorization, our metadata falls between or under both descriptive metadata and structural, as we are pulling from online resources—publications, videos, imagines—and also trying to describe the potential relationships between these resources. Regarding the quality of our metadata, both of us have highlighted the sensitivity of our topics in terms of their completeness, inclusivity, and ethical considerations. Specifically for me, in terms of completeness and trying to determine my own “golden minimum”, I think that one aspect of my struggle is constructing an inaccurate image of a person that I never knew, and that I can only ever know pieces of now that he is gone. Is there even a way to consider a minimum amount of information for such a situation? Or, should I conceive of the minimum in terms of what I am trying to convey about him? But then, I am also trying to consider his story through the lens of a larger issue, and contextualizing it within a wider frame of other people’s stories as well.

**Reply by Khushboobhutani** [reply]
I agree with Sidra's attempts to categorize our metadata into descriptive and structural metadata. Throughout this project, we have consistently been concerned about the intellectual property rights associated with such data and how not to breach it in the name of collective good given the sensitive nature of our projects. Therefore, trying to locate a "golden minimum" also becomes a little difficult at times provided that one wishes to present the same information from multiple angles in order to facilitate a well-rounded perspective. What further interested me was the idea of preservation metadata, a concern that I recently encountered when a major information source for my project was taken down a day before the presentation. Despite the omnipresence of the digital, the longevity of metadata is an important concern given the "controversial" status afforded to some information. Gilliland and Jessop in their articles further raise interesting concerns regarding metadata and its creation and accessibility. Currently, I am wondering about the ambiguities and differing opinions I have encountered while working on my project and if the maximization of contributions has enabled me to identify different roles that my metadata can perform.

**Comment by amj169** [comment]
I’m interested more specifically in how rights metadata practices might be practiced at say an institutional level, like here at Pitt. I’m continuously curious about how regulations surrounding the uses of technology in general are created (or not created). This aspect of talking about the data at the level of description makes me think further about people as data points and how that will be captured again and folded into this snapshot of metadata. I wonder about how an institution might talk about where they collect data from its population, and to what extent those data collection practices might be regulated or not. I remember talking briefly about ed tech systems and the ability to collect information for students about their use and engagement frequency like when they opened a certain document or how many times they opened the “start assignment” tab. I wonder if some of these collection practices are being disclosed (or if they have to be). I would imagine that even our Pitt IDs provide a significant amount of purchasing and location data. Every time I swipe onto the bus, it probably has my location. Maureen Whalen writes “Rights metadata can be made simple if everyone in the institution is aware of its long-term importance and there is a concerted, coordinated effort to research it, record it according to standards and best practices, and share it in fulfillment of the institutional mission in the digital age” and I wonder what happens when people either don’t recognize this or don’t really care to recognize this. Maybe I’m not focusing enough on the CC and copyright standard uses and perhaps I’ve wandered away from this entirely, but I’m always interested in the overlap of capitalism and data, and I’d like to hear more about how people are thinking about regulating some harmful practices when it comes to using peoples’ data for malicious? Intent or capital gains at a legislative level. In some ways, this feels more like a Borgman question. I found a YouTube video from the Berkman Klein Center for Internet and Society where Borgman talks about her “Data, data everywhere -- but how to manage and govern?” I’ll have to give it a watch!

**Comment by MustafaKandil** [comment]
This week's readings emphasized what metadata is, its structures, types, functions, creation methods, etc. I really like the definition of data about data; it's like a detailed map of data or objects. Jaffe's short guide and Gilliland's framework described it well. Especially in the context of archiving, museums, and libraries, they showed how data functions work. It made me feel like metadata is complex and a lifesaver. It makes the process of getting to know anything you're interested in easier. But, it also makes me think that there might be an ethical issue while compiling all components of data and also using this data. In Gilliland's text, it says, "Just as many members of the general public have participated in the development of web content, whether by blogging on Tumblr or by uploading photos onto Flickr or videos onto YouTube, they have also been creating, sharing, copying, and mapping metadata. Among the advantages of these developments is that individual web communities, such as affinity groups or hobbyists, may be able to create metadata that addresses their specific needs and vocabularies in ways that information professionals who apply metadata standards designed to cater to a wide range of audiences cannot." Especially in recent times, companies like Facebook, Google, and Instagram collect data about their users and use it for business purposes and other self-interested issues like politics and elections. Should there be limitations on collecting or using it?

Another thing that makes me think is the glossary of metadata standards. There are a lot of standards, and metadata is here to make it easy to get information, understand, or classify objects. When we have a lot of different standards, does it make the process harder rather than serving its functions?
